src/config/cmake/HYPRE_SetupHIPToolkit.cmake

```diff
@@ -200,6 +200,43 @@ if (HYPRE_ENABLE_LTO AND NOT MSVC)
   )
 endif ()
 
+# Check if user specified either WARP_SIZE or WAVEFRONT_SIZE
+if(DEFINED HYPRE_WAVEFRONT_SIZE)
```

Comment:
What's the reason for another name here (wavefront size instead of warp size)?

Reply:
To provide a more accurate proxy, in terms of naming, to hypre's warp size in the case of AMD GPUs.